website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -487,3 +487,419 @@ curl \
+`data.relationships.vars.data[].attributes.key`         | string  |           | The name of the variable.                                                                                                                                                                                             |
+`data.relationships.vars.data[].attributes.value`       | string  | `""`      | The value of the variable.                                                                                                                                                                                            |
+`data.relationships.vars.data[].attributes.description` | string  |           | The description of the variable.                                                                                                                                                                                      |
+`data.relationships.vars.data[].attributes.category`    | string  |           | Whether this is a Terraform or environment variable. Valid values are `"terraform"` or `"env"`.                                                                                                                       |
+`data.relationships.vars.data[].attributes.hcl`         | boolean | `false`   | Whether to evaluate the value of the variable as a string of HCL code. Has no effect for environment variables.                                                                                                       |
+`data.relationships.vars.data[].attributes.sensitive`   | boolean | `false`   | Whether the value is sensitive. If true then the variable is written once and not visible thereafter.                                                                                                                 |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "workspaces",
+    "relationships": {
+        "vars": {
+        "data": [
+          {
+            "type": "vars",
+            "attributes": {
+              "key": "region",
+              "value": "eu-central-1",
+              "category": "terraform",
+              "hcl": true,
+              "sensitive": false,
+            }
+          },
+          {
+            "type": "vars",
+            "attributes": {
+              "key": "ami",
+              "value": "ami‑077062",
+              "category": "terraform",
+              "hcl": true,
+              "sensitive": false,
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/organizations/hashicorp/no-code-modules/nocode-WGckovT2RQxupyt1/workspaces/ws-qACTToFUM5BvDKhC
+```
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "run-Cyij8ctBHM1g5xdX",
+    "type": "workspace-upgrade",
+    "attributes": {
+      "status": "planned",
```

Comment:
Is it possible to document an enumeration of what the valid values for `"status"` are? It seems important to be able to document clearly when someone should stop polling, and move on to the confirmation step.

This came up for me when I was QA testing -- and for whatever reason my workspace's status said `"cost_estimated"` and THAT was the waiting-for-approval step. I was confused because I was looking for `"planned"`.